File: JAVA/check_Prime.java
import java.util.Scanner;
class check_Prime{
  public static void main(String args[])
  {
    Scanner sc = new Scanner(System.in);
    System.out.println("ENTER A NUMBER:");
    int n = sc.nextInt();
    int c = 0;
    for(int i=1;i<=n;i++)
    {
      if(n%i==0)
      {
        c=c+1;
       }
    }
    if(c==2)
    {
     System.out.println("NUMBER IS PRIME.");
    }
    else
    {
     System.out.println("NUMBER IS NOT PRIME.");
     }
  }  
}

File: check_prime.cpp
#include<stdio.h>
int main(){
	int n,c=0,i;
	printf("ENTER A NUMBER:");
	scanf("%d",&n);
	for(i=1;i<=n ;i++)
	{
		if(n%i==0)
		{
			c+=1;
		}
	}
	if(c==2)
	{
		printf("NUMBER IS PRIME.");
	}
	else
	{
		printf("NUMBER IS NOT PRIME.");
	}
	return 0;
}
